src/release_linux.go

```diff
@@ -16,7 +18,14 @@ func writeDesktopFiles(mc *metaConfig) {
 }
 
 func configureAutologin(mc *metaConfig) {
-	// set up user autologin
+	if _, err := os.Stat("/usr/share/lightdm"); err == nil {
```

Comment:
We already have a `fileExists` function so use that here

Reply:
Status on this?

Reply:
Both works, so I am just gonna leave it

Reply:
Yea it's less clean tho; I'll just do it dw ab it